pkg/edgeview/src/network.go

```diff
@@ -0,0 +1,1134 @@
+// Copyright (c) 2021 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+
+	uuid "github.com/satori/go.uuid"
+
+	"github.com/grandcat/zeroconf"
+	"github.com/lf-edge/eve/pkg/pillar/types"
+)
+
+type urlStats struct {
+	recvBytes      int64
+	sentBytes      int64
+	sentNumber     int64
+}
+
+type appIPvnc struct {
+	ipAddr    string
+	vncEnable bool
+}
+
+func runNetwork(netw string) {
+	opts, err := checkOpts(netw, netopts)
+	if err != nil {
+		fmt.Println("runNetwork:", err)
+	}
+
+	for _, opt := range opts {
+		printTitle("\n === Network: <"+opt+"> ===\n\n", colorPURPLE, false)
+		var substring string
+		if strings.Contains(opt, "/") {
+			items := strings.SplitN(opt, "/", 2)
+			opt = items[0]
+			substring = items[1]
```

Comment:
This can contain arbitrary shell escapes. Can we constain the syntax of substring for the different network command?

Reply:
will do.